netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -60,7 +60,7 @@
 
   static final long MAX_CONNECTION_IDLE_NANOS_DISABLED = Long.MAX_VALUE;
   static final long MAX_CONNECTION_AGE_NANOS_DISABLED = Long.MAX_VALUE;
-  static final long MAX_CONNECTION_AGE_GRACE_NANOS_INFINITE = Long.MAX_VALUE;
```

Comment:
I'd much rather you leave all this as-is. Instead, only change secondGoAwayAndClose() to convert from this value to -1.

Reply:
No strong opinion there. Will do.